addons/udes_stock_picking_batch/models/stock_picking.py

```diff
@@ -148,3 +158,35 @@ def _priorities_has_ready_pickings(self, priorities, picking_type_id):
             ("state", "=", "assigned"),
         ]
         return self.search_count(domain) >= 1
+
+    def batch_to_user(self, user):
+        """Throws error if picking is batched to another user
+        or if user already has a batch
+        creates batch and adds picking to it otherwise
+        """
```

Comment:
Nit: can we rephrase it a bit? It's a little hard to follow. It should state clearly what the intentions are and not just what it does, then follow with the exceptions.

Also, I think the file is missing  a new line at the end? Might be wrong on this

Reply:
I don't see any complaint in github for a newline at the end of the file. I have also check in my editor, I think it is fine :)